.github/workflows/pages.yml

```
@@ -0,0 +1,60 @@
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+          fetch-depth: 0
+      - uses: jdx/mise-action@v2
+        with:
+          working_directory: ./cdviz-site/landingpage
+      - run: mise run build
+        with:
```

actionlint

this step is for running shell command since it contains at least one of "run", "shell" keys, but also contains "with" key which is used for running action

(syntax-check)